subworkflows/nf-core/variantcalling/haplotypecaller/main.nf

```diff
@@ -34,11 +34,10 @@ workflow RUN_HAPLOTYPECALLER {
         // group by interval
         genotype_gvcf_to_call = HAPLOTYPECALLER.out.vcf.join(HAPLOTYPECALLER.out.tbi).map{
         meta, gvcf, tbi ->
-            interval_name = meta.num_intervals > 1 ? (gvcf.simpleName - "${meta.id}_").replaceFirst("_",":") : meta.id
-            new_meta = [id: "joint_germline", interval_name: interval_name, num_intervals: meta.num_intervals]
+            interval_name = meta.num_intervals > 1 ? (gvcf.simpleName - "${meta.id}_").replaceFirst("_",":") : file(params.intervals).simpleName
```

Comment:
I would be very careful here. I am afraid this may lead to the weird resume errors/unmatching meta mpas we had before.

Reply:
hmm, would it be smarter to keep meta the same, and group py another (temporary) key?

Reply:
the problem is more the reqirval of the file name. I still can't explain this, but sometimes, when retrieving something from the file name like here, the name is incorrectly resolved later on (even though the matched file in the channel element is the correct one). In this case here this would lead to some very wrong results. (in the rest of sarek, since we group on patient ID it only lead to file name clashes, a) easy to find the bug, b) the actual output results were not impacted)